### PR TITLE
[13.0][MIG]stock_barcodes_move_location

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
TODO:
- [x] remove dependency to printer_zpl, added my mistake

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
@JoanSForgeFlow

**Review by JoanSForgeFlow — APPROVED** [review]
Tested on local using #263 and [stock_barcodes](https://github.com/ForgeFlow/stock-logistics-barcode/tree/13.0-mig-stock_barcodes).
Functional review, LGTM :+1:

**Comment by JoanSForgeFlow** [comment]
Tested using current OCA stock_barcodes, still working as expected

**Comment by LoisRForgeFlow** [comment]
Dependency #263 merged. Can you rebase?

**Comment by LoisRForgeFlow** [comment]
@AaronHForgeFlow can you check CI errors?

File: stock_barcodes_move_location/wizard/stock_barcodes_read_move_location.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+            self.move_location_id.stock_move_location_line_ids = [(4, line.id)]
+
+        self.move_location_qty = line.move_quantity
+
+    def check_done_conditions(self):
+        if self.product_id.tracking != "none" and not self.lot_id:
+            self._set_messagge_info("info", _("Waiting for input lot"))
+            return False
+
+        force_add_log = self.env.context.get("force_add_log", False)
```
Comment (AaronHForgeFlow) [inline stock_barcodes_move_location/wizard/stock_barcodes_read_move_location.py:119]: it seem this line was indented wrongly. Fixed now :)

**Comment by LoisRForgeFlow** [comment]
I think you still need to add a repo to oca-dependencies

```
odoo.exceptions.UserError: ("You try to install module 'stock_barcodes_move_location' that depends on module 'stock_move_location'.\nBut the latter module is not available in your system.", '')
```

**Comment by AaronHForgeFlow** [comment]
@LoisRForgeFlow you are correct. Thank you!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**